### ISSUE TITLE
Ansible inventory usage

### ISSUE THREAD

**Issue by stgrace** [stacklio/stackl#208]
# Description of feature 

Ansible inventory is hardcoded in Stackl-agent. This is not ideal

# Purpose of this new feature

The Ansible inventory should either be packaged as an Ansible collection or deployed once using the Stackl helm repository.

Both have pro's and cons.

Ansible collection:

Pro's:

- Each image has an ansible inventory coupled to it.

Cons:

- Ansible inventory is usually tied to the versioning of Stackl. 
    - In other words, it might not be benificial for each image to have its own inventory, since an upgrade to Stackl might break the inventory
    - All images need to be rebuild when a new inventory is needed

Helm:

Pro's

- Deployed as a CM with Stackl.
- Inventory should always work without having to rebuild images
- Fixes in inventory can be redeployed quickly

I personally prefer the Helm method
